scripts/github/release-artifacts.sh

```diff
@@ -161,7 +172,6 @@ while [ "$#" -gt 0 ]; do
     upload)
       shift
       COMMAND="upload"
-      shift
```

Comment:
Why is this shift not needed?

Reply:
Because it's not taking any arguments

Reply:
I removed the other shift ti make it clearer